### DISCUSSION TITLE
Composer install failed / conflicts

### DISCUSSION THREAD

**Discussion by mschweiger90** [coreshop/CoreShop#2513] [General]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

I use Pimcore 11.1.3 within PlatformVersion 2023.3. Pimcore is running with the current php Dockerimage: pimcore/pimcore:php8.2-debug-latest

When i require coreshop with composer (composer require coreshop/core-shop:^4.0) following happens:

![Bildschirmfoto vom 2024-01-16 08-49-21](https://github.com/coreshop/CoreShop/assets/17021488/144a92ca-842b-42e0-a05b-609e7c11a942)

Here is a second try with the option --with-all-dependencies

![Bildschirmfoto vom 2024-01-16 08-55-26](https://github.com/coreshop/CoreShop/assets/17021488/643c500c-0002-4e27-a10b-7c3d39b107f8)

Is the problem with these conflicts known? 
Thanks for the help, we look forward to using coreshop

**Comment by dpfaffenbauer** [comment]
can you try deleting composer.lock and your vendor folder and try again?

**Comment by mschweiger90** [comment]
Hi @dpfaffenbauer,
thx for your very fast response.
When i delete the composer.lock and vendor folder and try it again the same is happening:

![Bildschirmfoto vom 2024-01-16 09-23-13](https://github.com/coreshop/CoreShop/assets/17021488/2bf5c010-a1c9-461f-a7fe-457de03940f3)

I tried it also with a new installed Pimcore 11.1.3 inside the original pimcore docker container.

**Comment by dpfaffenbauer** [comment]
@mschweiger90 it probably comes down to CoreShop requiring `dachcom-digital/emailizr`, that requires `twig/inky-extra` and that requires a system extension `ext-xsl` which is not installed. can you try running it with `--ignore-platform-reqs`

**Reply by dpfaffenbauer** [reply]
@mschweiger90 could you solve the issue?

**Reply by gander** [reply]
@mschweiger90 The solution to this problem is to install `ext-xsl` or use the `--ignore-platform-reqs` method. I think this thread can be closed.

**Reply by dpfaffenbauer** [reply]
fixed by https://github.com/coreshop/CoreShop/pull/2584
